deprish_to_eia.py

```diff
@@ -0,0 +1,287 @@
+"""Connect the master unit list with depreciation records."""
```

Comment:
The module docstring will be used as the long-form usage message which is printed out when someone runs `deprish_to_eia.py --help` so it's a good idea to put something helpful here explaining what the script does in general.